src/main/java/teammates/ui/webapi/output/CourseData.java

```diff
@@ -9,16 +9,16 @@
 
     private final String courseId;
     private final String courseName;
-    private final String creationDate;
     private final String deletionDate;
     private final String timeZone;
+    private final long creationDate;
```

Comment:
I think the purpose of changing it to long is to ensure that it sorts correctly. Since `deletionDate` is not required for sorting so I thought that it should be fine. I guess we can see what @xpdavid thinks.

Reply:
You can change deletionDate to make the DTO consistent. After all, date formatting should be done in the frontend rather than backend since we adopt client-rendering.